src/ks-app/src/app/containers/buttons/buttons.component.html

```diff
@@ -0,0 +1,50 @@
+<h1>Buttons & Button Groups</h1>
+
+<p>Button groups are for creating collections of similar type action buttons. There are several styles of button
+</p>
+<nav style="margin-bottom: 24px">
```

Comment:
Would it make sense to create a custom class like `clr-example` or `clr-demo` (same as we do for our demo app) and add the custom styles (margin-bottom) here for example on that?

Reply:
I can make something like that. I think I was trying to keep it as vanilla as possible and opted for this approach to making presentation changes.